psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -863,7 +863,12 @@ bool from_string(T& t,
 void OEProp::compute()
 {
 
-    outfile->Printf( "\nProperties computed using the %s density matrix\n\n", title_.c_str());
+    
+    if (title_ == "")
```

Comment:
Please use brackets for if/else statements here.

Reply:
Thanks, will do. Is there a particular reason the if statements on lines 882-907 don't use brackets?

Reply:
Legacy semantics, trying to adhere a bit more to style guides these days.

Reply:
Makes sense, thanks!